syntax = "proto3";
package MSGID;

enum MsgID
{
    CS_HEART_BEAT       = 0; // 心跳
    CS_CONFIG_UPDATE    = 1; // 配置更新

    SC_NOTIFY           = 10000; // 下推
}